nixos/modules/system/boot/loader/grub/install-grub.pl

```diff
@@ -122,7 +122,7 @@ sub GetFs {
         my @superOptions = split /,/, $fields[$n + 2];
 
         # Skip the read-only bind-mount on /nix/store.
-        next if $mountPoint eq "/nix/store" && (grep { $_ eq "rw" } @superOptions) && (grep { $_ eq "ro" } @mountOptions);
```

Comment:
This might cause problems, if somebody uses a dedicated partition mounted on `/nix/store`.
Is there a smarter way to detect bind mounts?